src/backend/vulkan/mod.rs

```diff
@@ -317,19 +298,13 @@ impl Instance {
                         | vk::DebugUtilsMessageTypeFlagsEXT::PERFORMANCE
                         | vk::DebugUtilsMessageTypeFlagsEXT::VALIDATION,
                 )
-                .pfn_user_callback(Some(vulkan_debug_utils_callback))
-                .user_data(*logger_ptr as *mut _);
+                .pfn_user_callback(Some(vulkan_debug_utils_callback));
```

Comment:
Would be nice to have a span passed through the user data

Reply:
The idea was, that vulkan calls will likely be triggered through smithay functions, that will set the span anyway (which will then still be active, should the debug-callback be called from inside the vulkan driver). But I guess because we are exposing raw ash types, this might not always be the case?